code-style/git/README.md

```diff
@@ -6,18 +6,92 @@ We use Git version control for all of our projects at [Sparkbox][]. We host
+
+1. The reviewer may ask you to [rebase off master](#Rebasing-a-Branch-onto-the-Master-Branch) if the branch has diverged.
+
+1. Your reviewer will either approve or request changes on your pull request.
+    - If they request changes, please implement or further discuss their comments with them.
+
+1. If your reviewer approves your pull request, they will fast-forward merge your branch into master using the CLI.
+
+    ```cli
+    git merge --ff-only <branch-name>
+    ```
+
+1. The reviewer will then let you know your changes have been merged so you can move your Jira card.
+
+1. Once the PR is merged, the reviewer deletes the branch.
+
+
+## Naming Branches
+
+Consistent branch naming can help with team communication. A branch name might be the first peek a reviewer gets at what the work is related to. We use types (feat, fix, docs, etc.) to group our branch names so that we know what part of our workflow the branch belongs to, and a brief subject to describe what the branch does.
+
+`feat--onUrlChange-event`
+
+### Allowed Values for Types
+There are several types allowed to prefix branches and commits.
+
+- **feat** (new feature)
+- **fix** (bug fix)
+- **docs** (changes to documentation)
+- **style** (formatting, missing semi colons, etc; no code change)
+- **refactor** (refactoring production code)
+- **test** (adding missing tests, refactoring tests; no production code change)
+- **chore** (updating grunt tasks etc; no production code change)
+
+### Subject
+
+Subjects need to be short, but descriptive. Multi-word subjects are separated by a single hyphen between words.
+
+`feat--onUrlChange-event`
+
+`fix--unit-tests-IE9`
+
+### Branches with Multi-type Commits
+
+Even though our branches tend to be named similarly to our commit style, it's ok if you're working on a bug fix branch `fix--` and include other types of commits. For example, maybe you have a branch named `fix--unit-tests-IE9`, but you've done some `refactor`ing and made some `style` adjustments. Your commit messages for the branch might include:
+```
+fix: unit tests for IE9
+
+refactor: remove unnecessary conditionals
+
+style: add missing semi colons
+```
+
+It's totally ok to include all of those commits on your bug fix branch. You might have any combination of types in one branch if the work is related. We always leave the code better than we found it. There has to be a line somewhere, though. Pay attention if at any point your branch starts
+
+- addressing multiple components
+- feels too big/would overwhelm you if you were reviewing
+- could be broken up into more than one branch
+
+If you identify any of these concerns in your branch, it's worth exploring whether all the work you're doing should be reviewed together. Sometimes work does need to be all together, but in a lot of cases we can break up that work to make it easier for our code reviewers to provide a thorough review.
```

Comment:
Consider linking to this article https://www.netlify.com/blog/2020/03/31/how-to-scope-down-prs/

Reply:
Love it!